### PR TITLE
Functionality 6 change price

### PR DESCRIPTION
**Pull request by rkeynguyen**
it says a lot of files were changed so I'm not sure if I did this right. I added onto UserInterface and grocery store.

### CODE REVIEW

File: bin/.gitignore [hunk 1]
```diff
@@ -1,2 +1,2 @@
-/ui/
 /store/
+/ui/
```
Comment (carterclark) [inline bin/.gitignore]: I'm not sure what this does to the program?

**Review by carterclark — COMMENTED** [review]
Everything looks good. Could you explain the change to the gitignore?

File: src/store/facade/GroceryStore.java [hunk 2]
```diff
@@ -546,4 +546,24 @@ private Result reorderProduct(Product product) {
...
+	 * @return a result code that represents the outcome
+	 */
+	public Result changePrice(Request request) {
+		Result result = new Result();
+		Product product = productsList.searchById(request.getProductId());
+		product.setCurrentPrice(request.getProductCurrentPrice());
+
+		if (product.getCurrentPrice() == request.getProductCurrentPrice()) {
+			result.setResultCode(Result.ACTION_SUCCESSFUL);
+			System.out.printf("Product: %s, New Price: %.2f", product.getName(), product.getCurrentPrice());
```
Comment (carterclark) [inline src/store/facade/GroceryStore.java]: I really like this! We should make this format the standard instead of the concatenations.

Comment (paveldanek) [inline src/store/facade/GroceryStore.java]: Ricky, please be careful with this line (#562). The output to the console should be UI's job. GroceryStore is on the business side of the app. :)

**Review by carterclark — APPROVED** [review]

**Pull request merged**